lib/build.gradle.kts

```diff
@@ -0,0 +1,137 @@
+import cash.z.ecc.android.Deps
+import org.gradle.api.tasks.testing.logging.TestLogEvent
+
+plugins {
+    kotlin("multiplatform")
+    id("org.jetbrains.dokka") version "1.6.10"
+    id("io.kotest.multiplatform")
+    kotlin("plugin.serialization")
+    id("dev.icerock.mobile.multiplatform-resources")
+    id("com.vanniktech.maven.publish") version "0.19.0"
+}
+
+group = project.property("GROUP")!!
+version = project.property("VERSION_NAME")!!
+
+
+kotlin {
+    jvm {
+        compilations.all {
+            kotlinOptions.jvmTarget = "11"
+        }
+        withJava()
+    }
+    js(IR) {
+        browser() // to compile for the web
+        nodejs() // to compile against node
+    }
+    macosX64()
+    macosArm64()
+    iosX64()
+    iosArm64()
+//    linuxX64()
+
```

Comment:
I don't know if I agree with that haha. A build or runtime issue with a native target won't prevent jvm or js from being built. The overall build might break, but each target can always be built individually with their respective gradle tasks. Just some food for thought

Reply:
It might be avoidable now by explicitly invoking a task, but it'll be much harder once dependency locking is enabled since we'll need a `resolveAll` task that makes it hard to run a specific task only e.g. like this https://github.com/zcash/zcash-android-wallet-sdk/blob/master/build-conventions/src/main/kotlin/zcash.dependency-conventions.gradle.kts

This was a breaking issue with the Kotlin 1.6.20 preview releases recently, so its top of mind in terms of things I've seen happen.